test/spec/unit/pbjs_api_spec.js

```diff
@@ -720,7 +719,7 @@ describe('Unit: Prebid Module', function () {
       ajaxStub.restore();
     });
 
-    it('should get correct hb_pb with cpm between 0 - 5', function () {
+    it('should get correct ' + CONSTANTS.TARGETING_KEYS.PRICE_BUCKET + ' with cpm between 0 - 5', () => {
```

Comment:
Can you change back the various `() => {` references to the `, function() {` variant?  These were implemented to make the unit tests more compliant with Mocha.

Reply:
Hello @jsnellbaker , 
sure, let me check.

Reply:
Hello @jsnellbaker ,
I have made these changes.